src/sentry/search/events/datasets/discover.py

```diff
@@ -256,6 +266,20 @@ def function_converter(self) -> Mapping[str, SnQLFunction]:
                     ),
                     default_result_type="percentage",
                 ),
+                SnQLFunction(
+                    "group_array",
+                    required_args=[
+                        ColumnTagArg("column"),
+                    ],
+                    snql_aggregate=lambda args, alias: CurriedFunction(
+                        "groupArray",
+                        [100],  # max size of 100, doesnt need to be configurable for now
```

Comment:
Because this is common code, people will come to depend on it in the future.  Is there a way for us to pass this value explicitly in the caller?  If not I would suggest renaming to `group_array_max_100`.  We're making assumptions about the maximum number of replay-ids being returned elsewhere in the code base.  If this value is altered by a future party it could create an outage.

Reply:
I looked again this AM and figured out how to pass it in (needed to convert the argument to integer w/ NumericRange). Will defer to the QueryBuilder experts but I think how I did it is okay. thanks for the nudge here!